src/main/java/apoc/diff/Diff.java

```diff
@@ -0,0 +1,110 @@
+package apoc.diff;
+
+import apoc.Description;
+import apoc.result.DiffResult;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.procedure.*;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author Benjamin Clauss
+ * @since 29.03.2018
+ */
+public class Diff {
+
+    @Context
+    public GraphDatabaseService db;
+
+    @UserFunction
+    @Description("apoc.diff.maps([leftMap],[rightMap])" )
```

Comment:
please add an explanation to all descriptions ?

Reply:
it's not really a diff, it's just an equals, which is the same as  `WHERE leftMap = rightMap` in cypher, or?
I thought you'd have a function that outputs the differences between both, which then should go to `apoc.map.*`